apps/api/src/domain/models/bibliographic-reference/common/commands/register-digital-representation-of-bibiliographic-citation/register-digital-representation-of-bibliographic-ciation.command.ts

```diff
@@ -0,0 +1,32 @@
+import { ICommandBase } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType } from '@coscrad/data-types';
+import { DigitalTextCompositeId } from '../../../../digital-text/commands';
+import { BibliographicReferenceCompositeIdentifier } from '../../../shared/BibliographicReferenceCompositeIdentifier';
+
+@Command({
+    type: `REGISTER_DIGITAL_REPRESENTATION_OF_BIBLIOGRAPHIC_CITATION`,
+    label: `Add Digital Content for Citation`,
+    description: `Register a Digital Text as the subject of a Bibliographic Citation`,
```

Comment:
`subject` or `digital representation`?  Just curious about terminology.

Reply:
Yea. It's a tough one. I suppose we want to expose the "digital representation" terminology to users. I thought maybe we didn't. Let's talk about this. For now, we are not exposing the command in the UX anyway.